packages/editor/src/components/rich-text/index.js

```diff
@@ -88,6 +90,14 @@ export class RichText extends Component {
 			this.multilineWrapperTags = [ 'ul', 'ol' ];
 		}
 
+		if ( this.props.onSplit ) {
+			deprecated( 'wp.editor.RichText onSplit prop', {
```

Comment:
We need to include it in both the changelog of `editor` package (not quite sure if this is major version increment). We have to put `4.4` in here. Otherwise, linter will fail build when we bump version of the package to 4.2.

We should also include it on the deprecations list.